Builds fail bc asmdef does not allow unsafe code

https://github.com/leapmotion/UnityModules/blob/35e6985a55aef5825b8f06bf0c40e418ab63fa8d/Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef#L12

```
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(28,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(46,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(64,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(84,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(104,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(128,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(152,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(172,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(200,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(223,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(246,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(271,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(296,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(325,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(354,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(379,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(405,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(424,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(443,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(464,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(485,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(510,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(535,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(556,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(584,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(607,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(630,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(655,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(680,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(709,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(738,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
##[error]Assets\Plugins\LeapMotion\Core\Scripts\Utils\BitConverterNonAlloc.cs(763,7): error CS0227: Unsafe code may only appear if compiling with /unsafe. Enable "Allow 'unsafe' code" in the inspector for 'Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef' to fix this error.
[ScriptCompilation] Recompiling all scripts because: Recompiling scripts for editor because player build failed.
DisplayProgressNotification: Build Failed
##[error]Error building Player because scripts had compiler errors

```

--- a/Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef
+++ b/Assets/Plugins/LeapMotion/Core/Scripts/LeapMotion.asmdef
@@ -9,7 +9,7 @@
     ],
     "includePlatforms": [],
     "excludePlatforms": [],
-    "allowUnsafeCode": false,
+    "allowUnsafeCode": true,
     "overrideReferences": true,
     "precompiledReferences": [
         "nunit.framework.dll"

